docs/source/gallery/inference_diagnostics/10_plot_pair.py

```diff
@@ -0,0 +1,29 @@
+"""
+# Scatter plot of all variables against each other with divergences
+
+Plot all variables against each other in the dataset.
+
+---
+
+:::{seealso}
+API Documentation: {func}`~arviz_plots.plot_pair`
+:::
+"""
+import numpy as np
+from arviz_base import load_arviz_data
+
+import arviz_plots as azp
+
+azp.style.use("arviz-variat")
+
+dt = load_arviz_data("centered_eight")
+pc = azp.plot_pair(
+    dt,
+    var_names=["theta", "mu"],
+    coords={"school": ["Lawrenceville", "Mt. Hermon"]},
+    visuals={"divergence":True},
+    marginal=True,
+    marginal_kind="kde",
```

Comment:
maybe change this so the plot looks more different than the one in the distributions section

```suggestion
    marginal=False,
```